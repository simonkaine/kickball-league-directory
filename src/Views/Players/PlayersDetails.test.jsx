import { render, screen } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import PlayerDetails from "./PlayersDetails";

it('Should test for detailed view of player', async () => {
    const {container} = render(
        <MemoryRouter>
            <PlayerDetails match={{ params: {playerId: '1'} }}/>
        </MemoryRouter> 
    );
    
    screen.getByText('...Loading player');
    const thePlayerName = await screen.findByText(/Simon Kaine/i);
    expect(thePlayerName).toBeInTheDocument()
    expect(container).toMatchSnapshot();
});