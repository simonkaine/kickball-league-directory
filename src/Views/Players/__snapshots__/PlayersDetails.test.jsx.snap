// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test for detailed view of player 1`] = `
<div>
  <div
    class="PlayerDetails"
    style="border: 1px solid black; padding: 100px 0px 0px 0px; height: 100vh;"
  >
    <h1
      style="font-family: Century Gothic; font-size: 4em;"
    >
      Ben E. Jetts
    </h1>
    <p
      style="font-size: 2em;"
    >
      Pitcher
       for the 
      Bridge City Sneakers
    </p>
  </div>
</div>
`;
